### PR TITLE
Improve HTTP fingerprints for more cameras, AirTunes, DVRs, Splunk, Oracle XML DB, more

### PR DESCRIPTION
**Pull request by jhart-r7**
## Description

Various improvements and additions to (mostly) HTTP fingerprints. 

## Motivation and Context

More fingerprints!

## How Has This Been Tested?

Examples, `bundle exec rspec` passes

## Types of changes
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have updated the documentation accordingly (or changes are not required).
- [x] I have added tests to cover my changes (or new tests are not required).
- [x] All new and existing tests passed.

### CODE REVIEW

File: xml/http_cookies.xml [hunk 1]
```diff
@@ -138,7 +138,7 @@
     <example>NSC_AAAC=xyz;</example>
     <param pos="0" name="os.vendor" value="Citrix"/>
     <param pos="0" name="os.family" value="NetScaler"/>
-    <param pos="0" name="os.device" value="NetScaler"/>
+    <param pos="0" name="os.device" value="Network management device"/>
```
Comment (tsellers-r7) [inline xml/http_cookies.xml]: Existing convention is leading caps.
```
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Network Management Device"/>
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Network Management Device"/>
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Network Management Device"/>
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Network Scanner"/>
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Network Management Device"/>
snmp_sysdescr.xml:    <param pos="0" name="os.device" value="Network Management Device"/>
telnet_banners.xml:    <param pos="0" name="os.device" value="Network Management Device"/>
```

File: xml/http_servers.xml [hunk 2]
```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <fingerprints matches="http_header.server" protocol="http" database_type="service" preference="0.90">
   <!-- HTTP Server headers are matched against these patterns to fingerprint HTTP servers. -->
+  <fingerprint pattern="(?i)^AirTunes/([\d\.]+)$">
+    <description>Apple AirTunes/AirPlay, more generally RTSP used by a variety of wireless a/v products</description>
+    <example service.version="220.68">AirTunes/220.68</example>
+    <param pos="0" name="service.vendor" value="Apple"/>
+    <param pos="0" name="service.product" value="AirTunes"/>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="hw.device" value="Media Server"/>
```
Comment (tsellers-r7) [inline xml/http_servers.xml]: At some point we should look at if these are source or sinks as well as if this banner can be seen on Macs or Windows boxes running iTunes, Kodi, etc.

Comment (jhart-r7) [reply xml/http_servers.xml]: My understanding is that this banner will be shown by any device speaking the protocol used by AirTunes, which is largely iTunes/other apple stuff but will definitely also include other vendors/products.  That is why I kept it generic; we know the protocol and can suggest something about the device.

File: xml/http_cookies.xml (hunk 1 above)
Comment (jhart-r7) [reply xml/http_cookies.xml]: Resolved for consistency within recog.

**Pull request merged**